README.md

```diff
@@ -26,29 +26,43 @@ Also the pre-prod.pem key must exist in this directory.
 
 1. Ensure you have been set up with an AWS account
 
-2. Obtain your AWS credentials - http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-set-up.html#cli-signup
+1. Obtain your AWS credentials - http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-set-up.html#cli-signup
 
-3. And configure them - http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html
+1. And configure them - http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html
 
 ## Setting up Terraform
 
 1. Install terraform(terraform.io) and add the binary to your shell path. If you are using Homebrew:
 
-  `brew install terraform`
+```
+brew install terraform
+```
 
-2. Copy `terraform.tfvars.example` to `terraform.tfvars` in both author and survey-runner. You'll need to change all values containing an 'X' to match your requirements, including the AWS credentials you set up previously.
+1. Generate ssh key pair in AWS for your AWS user
 
-3. To deploy survey-runner:
+1. Download and restrict access by executing:
 
-  - Export an `AWS_ENVIRONMENT_NAME` environment variable e.g. `export AWS_ENVIRONMENT_NAME=preprod`
+```
+chmod 400 mykey.pem
+```
+
+1. Move the ssh key to `eq-terraform/survey-runner`
+
+1. Copy `terraform.tfvars.example` to `terraform.tfvars` in both author and survey-runner. You'll need to change all values containing an 'X' to match your requirements, including the AWS credentials you set up previously.
+
+1. To deploy survey-runner:
+
+  - Export an `AWS_ENVIRONMENT_NAME` environment variable e.g. `export AWS_ENVIRONMENT_NAME=preprod` or `export AWS_ENVIRONMENT_NAME=$USER`
+
+  - If using awscli profile export an `AWS_DEFAULT_PROFILE` environment variable e.g. `export AWS_DEFAULT_PROFILE=ons`
```

Comment:
probably need to add a note that this step is only required if running multiple AWS profiles